server/src/main/java/org/jboss/as/server/operations/NativeManagementAddHandler.java

```diff
@@ -75,6 +75,8 @@ protected void performRuntime(OperationContext context, ModelNode operation, Mod
 
         final ServiceName endpointName = ManagementRemotingServices.MANAGEMENT_ENDPOINT;
         final String hostName = WildFlySecurityManager.getPropertyPrivileged(ServerEnvironment.NODE_NAME, null);
+
+        NativeManagementServices.installManagementWorkerService(serviceTarget, context.getServiceRegistry(false));
```

Comment:
I followed the same piece of code used in the controller version: org.jboss.as.host.controller.operations.HttpManagementAddHandler#installHttpManagementServices. 

In the controller version, before install the xnio mgmt worker first it checks if the service is not already registered in the ServiceRegistry. I used the same logic here in the server version. That was the reason of line 79, I call NativeManagementServices.installManagementWorkerService method there as the controller version does.

So, if I understand you well, you suggest to remove the call in line 79 and call directly to ManagementWorkerService.installService, am I right?